src/components/sidebar/networkView/nodeView/NetworkNode.tsx

```diff
@@ -101,7 +150,12 @@ class NetworkNode extends React.Component<INetworkNodeProps, InetworkNodeState>
             <div className={s.networkNodeView}>
                 <div className={s.content}>
                     <ViewHeader
-                        closePromptMessage={undefined}
+                        closePromptMessage={
+                            this.props.nodeStore!.isDirty ? message : undefined
+                        }
+                        showEditButton={true}
+                        isEditing={!isEditingDisabled}
+                        onEditButtonClick={this.onEditButtonClick}
```

Comment:
Rename onEditButtonClick = toggleIsEditingDisabled (everywhere else as well)

(name describing what function does is better than name a name of "what happened")